mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -201,10 +201,7 @@ def _expand_vars(params: Dict[str, TunableValue], global_config: Dict[str, Tunab
         """
         Expand `$var` into actual values of the variables.
         """
-        return {
-            key: Template(val).safe_substitute(global_config) if isinstance(val, str) else val
-            for key, val in params.items()
-        }
+        return DictTemplater(params).expand_vars(extra_source_dict=global_config)
```

Comment:
Left in a simple staticmethod due to existing test infra usage.

Reply:
makes perfect sense